### PR TITLE
update release notes for app config and search

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: releases/2020-09/python.md [hunk 1]
```diff
@@ -15,10 +15,12 @@ The Azure SDK team is pleased to make available the September 2020 client librar
 #### Updates
 
 - Key Vault Certificates
+- App Configuration
 
 #### Preview
 
-- Azure Event Grid preview 1 (2.0.0b1)
+- Azure Event Grid preview 1
+- Azure Search Documents preview 2
```
Comment (iscai-msft) [inline releases/2020-09/python.md:23]: for both event grid and search, should we be including the version up here?

**Review by iscai-msft — APPROVED** [review]
Left a comment, but overall approve

File: releases/2020-09/python.md (hunk 1 above)
Comment (xiangyan99) [reply releases/2020-09/python.md:23]: Usually we don't add this information here.

Comment (iscai-msft) [reply releases/2020-09/python.md:23]: can you remove it

Comment (xiangyan99) [reply releases/2020-09/python.md]: Seems like I interpreted correctly. ;)

**Pull request merged**